modules/ROOT/pages/map-users-roles-ldap-task.adoc

```diff
@@ -1,39 +1,151 @@
+
+----
+<ns2:Attribute Name="Groups" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified">
+  <ns2:AttributeValue>Mule_Org_Admin_XXX</ns2:AttributeValue>
+  <ns2:AttributeValue>Mule_Exchange_XXX</ns2:AttributeValue>
+</ns2:Attribute>
+----
+
+=== Map SAML Users to Roles
 
 . In Anypoint Platform, navigate to Access Management.
 . Click *Roles*.
 . Click *Add Role* to create a role for each group of users in your organization.
-+
-image::external-identity-34af9.png[]
-
 . Specify a role name and description.
-. Click *Add Role* to add the role:
-+
-image::external-identity-c731b.png[]
+. Click *Add Role* to add the role.
+. In the *Roles* section, click the name of the new role:
+. Click *Set external group mapping*.
+. Enter the your array of SAML groups.
+. Click *Set names*.
 +
+To map more than one attribute name to the selected role, click *Add More* and add another attribute.
+
+. Repeat this process for each role that you want to map to an external group.
+
+== Configure Multiple Group Mappings for Single Sign-On using OpenID Connect (OIDC)
+
+External identity providers format group mappings in different ways.
+Anypoint Platform checks the `UserInfo` endpoint for user information. If no user information is found, it then checks the `IdToken` endpoint.
 
+Okta, Auth0, Salesforce, Google, and Microsoft ADFS return groups that can be used for OIDC group mappings.
+You can use other identity providers if they call the `UserInfo` or `IdToken` endpoints.
+
+=== Map OIDC Group Users to Roles
+
+. Obtain a response to the `UserInfo` or `IdToken` endpoints with a group claim from your external identity provider.
+. In Anypoint Platform, navigate to Access Management.
+. Click *External Identity*.
+. From *Identity Management*, select OIDC.
+. Verify that the value in the *Group Scope* field matches the value in your IDP group expression.
+. Verify that the value in the *Group Attribute JSONata Expression* field matches the value from your JSONata query of your IDP group expression.
+. Click *Roles*.
+. Click *Add Role* to create a role for each group of users in your organization.
+. Specify a role name and description.
+. Click *Add Role* to add the role.
 . In the *Roles* section, click the name of the new role:
+. Click *Set external group mapping*.
+. Enter the group to which you want to map.
+. Click *Set names*.
 +
-image::external-identity-35f9a.png[]
-+
+To map more than one attribute name to the selected role, click *Add More* and add another attribute.
 
-. Click *Set external group mapping*:
-+
-image::external-identity-251b8.png[]
-+
+=== Examples of OIDC Group Expressions
+
+The following examples show responses `UserInfo` or `IdToken` endpoints that contain group claims from different external identity providers.
+
+==== Okta
 
+[source,xml
+----
+{
+ ...
+  "groups": [
+    "Everyone",
+    "groupOne"
+  ]
+}
+----
```

Comment:
can we add another block that has the example corresponding JSONata expression
this one would be 
`groups`